config.rb

```diff
@@ -71,3 +71,7 @@
   # Minify Javascript on build
   # activate :minify_javascript
 end
+
+activate :disqus do |d|
+  d.shortname = 'talksdiscoverbsdcom'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.